src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc

```diff
@@ -1,27 +1,21 @@
 [id="configuring-namespace-strategies_{context}"]
 = Configuring namespace strategies
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+Word 'namespace' is used interchangeably with OpenShift's 'project'.
 
-[discrete]
-== Prerequisites
+=== Namespace per Workspace strategy
+TODO
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+=== All workspaces in one namespace strategy
+TODO
 
-[discrete]
-== Procedure
+=== One Namespace per User strategy
+This strategy provides more security than ‘One namespace for all workspaces strategy’, by isolating user to his/her own namespace. At the same time it’s not so resource heavy as ‘Namespace per workspace strategy’.
 
-. Start each step with an active verb.
+This strategy can be configured by setting `CHE_INFRA_KUBERNETES_NAMESPACE` on Kubernetes, or respectively `CHE_INFRA_OPENSHIFT_PROJECT` on OpenShift. Value must contain one or more user placeholders. Currently supported placeholders are `<username>` and `<userid>`. By this, namespace name will be evaluated different for each user, but same for all workspaces by one user.
 
-. Include one command or action per step.
+Example:
+To have namespaces named like `che-ws-user1`, `che-ws-user2`, `…` set `CHE_INFRA_KUBERNETES_NAMESPACE=che-ws-<username>`.
 
-. Use an unnumbered bullet (*) if the procedure includes only one step.
-
-[discrete]
-== Additional resources
-
-* A bulleted list of links to other material closely related to the contents of the procedure module.
-* For more details on writing procedure modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
-* Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
+Note:
+Compatible PVC strategies are `unique`, `common` and `perWorkspace`
```

Comment:
Well, a common PVC strategy is compatible but it would mean that only 1 workspace must be run at the same time. Maybe it makes sense to clarify it here.

Reply:
And do you know some incompatible namespace and PVC strategies?

Reply:
> Well, a common PVC strategy is compatible but it would mean that only 1 workspace must be run at the same time. Maybe it makes sense to clarify it here.

updated.


> And do you know some incompatible namespace and PVC strategies?

I don't know. TBH I'm not sure I know all namespace and all PVC strategies.